src/test/suite/fixtures/recorded/navigationMap/chuckHarpPastAirTakeWhale.yml

```diff
@@ -0,0 +1,49 @@
+spokenForm: chuck harp past air take whale
+languageId: plaintext
+command:
+  actionName: remove
+  partialTargets:
+    - type: range
+      start:
+        type: primitive
+        mark: {type: decoratedSymbol, symbolColor: default, character: h}
+      end:
+        type: primitive
+        mark: {type: decoratedSymbol, symbolColor: default, character: a}
+      excludeStart: false
+      excludeEnd: false
+  extraArgs: []
+initialState:
+  documentContents: hello world whatever
+  selections:
+    - anchor: {line: 0, character: 20}
+      active: {line: 0, character: 20}
+  marks:
+    default.h:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 5}
+    default.a:
+      start: {line: 0, character: 12}
+      end: {line: 0, character: 20}
+    default.w:
+      start: {line: 0, character: 6}
+      end: {line: 0, character: 11}
+finalState:
+  documentContents: ""
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  marks:
+    default.h:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 0}
+    default.a:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 0}
+    default.w:
+      start: {line: 0, character: 0}
+      end: {line: 0, character: 0}
```

Comment:
Notice how clobbering the range just leaves the mark at the beginning of the deleted range